mParticle-Google-Analytics-Firebase-GA4/MPKitFirebaseGA4Analytics.h

```diff
@@ -14,6 +14,9 @@
 @property (nonatomic, unsafe_unretained, readonly) BOOL started;
 @property (nonatomic, strong, nullable) MPKitAPI *kitApi;
 
++ (void)setCustomNameStandardization:(NSString*_Nonnull(^_Nullable)(NSString* _Nonnull name))standardization;
++ (NSString*_Nonnull(^_Nullable)(NSString* _Nonnull name))customNameStandardization;
```

Comment:
```suggestion
+ (void)setCustomNameStandardization:(NSString * _Nonnull (^_Nullable)(NSString * _Nonnull name))standardization;
+ (NSString * _Nonnull (^_Nullable)(NSString * _Nonnull name))customNameStandardization;
```

Minor readability tweak (man obj-c block syntax is so gross lol)